m2m-transfers/m2m-transfers-json-contract.md

```diff
@@ -134,54 +131,99 @@ list of institution and member limits, eligible shares, list of scheduled transf
   "loggingErrorMessage": "No eligible shares found"
 }
 ```
-## PROCESS REQUEST STATE
-### Client Request (PROCESS)
+## CREATE TRANSFER REQUEST STATE
+### Client Request (CREATETRAN)
 
 ```json
 {
-  "rgState": "PROCESS",
+  "rgState": "CREATETRAN",
   "powerOnFileName": "BANNO.M2MTRANSFERS.V1.POW",
   "userChrList": [
-    { "id": 1, "value": "[request type]:[data]|[data]|[data]|[data]|[data]|[data]" },
-    { "id": 2, "value": "[data]|[data]|[data]|[data]|[data]|[data]" },
-    { "id": 3, "value": "" },
-    { "id": 4, "value": "" },
+    { "id": 1, "value": "395" }, // recipientLoc if available, empty if not
```

Comment:
Just as an FYI, the locator code is a number which could be stored in the @RGUSERNUM1 or USERNUM2 field, thus freeing up this character variable  for other uses. For example, we're already talking with a CU who will want a custom version which would allow the creation of a mailing name record in addition to the primary name thereby requiring the additional @USERCHAR fields.